src/sentry/static/sentry/app/components/charts/utils/getColorPalette.tsx

```diff
@@ -0,0 +1,13 @@
+import {Theme} from 'app/utils/theme';
+
+/**
+ * Constructs the color palette for a chart given the Theme and optionally a
+ * series length
+ */
+export function getColorPalette(theme: Theme, seriesLength: number | undefined | null) {
```

Comment:
```suggestion
export function getColorPalette(theme: Theme, seriesLength?: number | null) {
```

Reply:
I think this is semantically different. One allows you to pass the series length or not, the other requires that the argument be there

Reply:
yes, that is true, but in the end, the result is the same 🤷‍♀️

Reply:
👍

Reply:
Fixed